WordPressKitTests/StatsRemoteV2Tests.swift

```diff
@@ -364,7 +365,35 @@ class StatsRemoteV2Tests: RemoteTestCase, RESTTestable {
 
         waitForExpectations(timeout: timeout, handler: nil)
     }
-  
+
+    func testFetchDownloadsData() {
+
+        // TODO: update with real data once endpoint is live
+
+        let expect = expectation(description: "It should return file download data for a month")
+        
+        let dateComponents = DateComponents(year: 2019, month: 7, day: 29)
+        let date = Calendar.autoupdatingCurrent.date(from: dateComponents)!
+
+        stubRemoteResponse(siteDownloadsDataEndpoint, filename: getDownloadsDataFilename, contentType: .ApplicationJSON)
+
+        remote.getData(for: .month, endingOn: date) { (fileDownloads: StatsFileDownloadsTimeIntervalData?, error: Error?) in
+            XCTAssertNil(error)
+            XCTAssertNotNil(fileDownloads)
+
+            XCTAssertEqual(fileDownloads?.otherDownloadsCount, 0)
+            XCTAssertEqual(fileDownloads?.totalDownloadsCount, 0)
+
+            XCTAssertEqual(fileDownloads?.fileDownloads.count, 0)
+//            XCTAssertEqual(fileDownloads?.fileDownloads.first!.file, "")
+//            XCTAssertEqual(fileDownloads?.fileDownloads.first!.downloadCount, 666)
```

Comment:
Do we need these tests? If not can we delete these comments?

Reply:
Hey @danielebogo . I left these in so I wouldn't forgot to add them back in later. I'll add a `TODO` explaining why they are there. 🤞 they'll be updated before this is released.